beam-postgres/Database/Beam/Postgres/Conduit.hs

```diff
@@ -115,77 +177,119 @@ executeStatement conn x =
     syntax <- pgRenderSyntax conn x
     Pg.execute_ conn (Pg.Query syntax)
 
+
+-- | Runs any query that returns a set of values
+runQueryReturning'
+  :: ( C.MonadResource m, Fail.MonadFail m, FromBackendRow Postgres r )
+  => Pg.Connection -> PgSyntax
+  -> CONDUIT_TRANSFORMER () r m ()
+runQueryReturning' (conn@Pg.Connection {..}) x = do
```

Comment:
Personally, I'd advise against `RecordWildCards` since it makes it impossible to know what's bound here without knowing the definition of `Pg.Connection`.